test/cpp/rpc/CMakeLists.txt

```diff
@@ -2,15 +2,12 @@ set(TORCH_RPC_TEST_DIR "${TORCH_ROOT}/test/cpp/rpc")
 set(TORCH_RPC_TEST_SOURCES
   ${TORCH_ROOT}/test/cpp/common/main.cpp
   ${TORCH_RPC_TEST_DIR}/test_wire_serialization.cpp
-  ${TORCH_RPC_TEST_DIR}/test_tensorpipe_serialization.cpp
+  ${TORCH_RPC_TEST_DIR}/test_e2e_process_group.cpp
```

Comment:
Can you double-check this, from when you were merging?
I think you probably didn't intend to remove the tensor pipe targets?